integrations/rust/install-driver.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+rm -rf ~/.rustup
+curl https://sh.rustup.rs -sSf | sh -s -- -y --no-modify-path
```

Comment:
should we be installing rustup and cargo in the project directory rather than home directory, similar to what we switched to for the driver?

Reply:
I think it's not strictly necessary in the way that it was for the driver -- it came up there because the documentation task does some chmod-ing to simulate a read-only file system, which can break future builds that use the same cargo install. 
that said, per Evergreen team it's considered best practice to not write to the home directory on the hosts since they do get reused:

> Best practice for evergreen is to stick all files in the task working directory rather than writing to the home directory